student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] `parent_permission` is a parent attribute and is initialised by the `super` function, you don't need that assignment but just the parameter `parent_permission`